### PR TITLE
Update gzip sizes

### PR DESCRIPTION
**Pull request by alancutter**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ module.exports = function(grunt) {
           global_defs: defines,
           dead_code: false
         },
-        mangle: false
+        mangle: false,
+        report: 'gzip',
```
Comment (suzyh) [inline Gruntfile.js]: Does the trailing comma cause any problems? I haven't yet learnt the JS syntax requirements here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,23 +33,25 @@
...
 
-3.  Specify the new version inside `package.json` (for NPM), for example:
+1.  Specify the new version inside `package.json` (for NPM), for example:
 
     ```js
       "version": "3.13.37",
     ```
 
-4.  Submit both changes to web-animations-next then follow the procedure to push from web-animations-next to web-animations-js.
+1.  Build the polyfill with `npm install && grunt` then update `README.md`'s Build Target Comparison with the current gzipped sizes.
```
Comment (suzyh) [inline CONTRIBUTING.md]: Do we have guidelines about what the acceptable upper bound on gzipped size is?

**Comment by suzyh** [comment]
Please resolve the conflicts.

File: Gruntfile.js (hunk 1 above)
Comment (alancutter) [inline Gruntfile.js]: It's allowed in Javascript but not JSON.

File: CONTRIBUTING.md (hunk 2 above)
Comment (alancutter) [inline CONTRIBUTING.md]: Not sure, we should confirm with Polymer animation team about this. The focus on size originated from their desire to use web-animations-next-lite.

**Comment by alancutter** [comment]
Rebased.

File: CONTRIBUTING.md (hunk 2 above)
Comment (suzyh) [inline CONTRIBUTING.md]: @dstockwell your thoughts?

Comment (alancutter) [inline CONTRIBUTING.md]: I don't think establishing a guideline should block this patch.

Comment (suzyh) [inline CONTRIBUTING.md]: Good point. lgtm

**Pull request merged**